pages/index.tsx

```diff
@@ -22,6 +24,11 @@ import AllPositions from "../features/all-positions/AllPositions";
 import Weth from "../features/weth/Weth";
 import Yield from "../features/yield/Yield";
 
+import GitHubIcon from "@material-ui/icons/GitHub";
+import DiscordIcon from "../public/discord.svg";
+import TwitterIcon from "../public/twitter.svg";
```

Comment:
I'm not really a fan of having to import svg icons. ideally, we should only use stuff from `@material-ui/icons/GitHub`. 

Twitter has an icon:
```
import TwitterIcon from '@material-ui/icons/Twitter';
```

Discord doesn't have one though so I guess we'll have to use that from an SVG.

Reply:
We shouoldn't mix and match SVG vs MUI Icon as they look different, so I propose we either move all to SVG or all to MUI Icon (and remove discord)

Reply:
I'm going to go ahead and remove discord as the twitter bio has a discord link

Reply:
I removed it it looks much better now PTAL

Reply:
That sounds good. At some point down the road, we might want to have some sort of link somewhere else on the page:
```
Having issues? Use our [DISCORD_LINK] to get help!"
```